hillman_pipeline/experiment.py

```diff
@@ -88,9 +88,9 @@ class Organ(dj.Lookup):
 @schema
 class Session(dj.Manual):
     definition = """
-    -> Species
-    session_start_time      : datetime
+    session_name            : varchar(128)
```

Comment:
Consider using a shorter attribute since the session name will be copied into many other tables. Would it be possible to use something like `session_code` that would be 12 or 16 characters max?